From: Ilya Dryomov <idryomov@gmail.com>
Date: Fri, 20 Nov 2015 22:16:46 +0100
Subject: block: rename request_queue slab cache
Git-commit: c2789bd403f4c0c541a359cf318b1dda9a14234f
Patch-mainline: v4.5-rc1

Name the cache after the actual name of the struct.

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.com>
---
 block/blk-core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/blk-core.c b/block/blk-core.c
index 5131993b..c0c884e 100644
--- a/block/blk-core.c
+++ b/block/blk-core.c
@@ -3513,7 +3513,7 @@ int __init blk_dev_init(void)
 	request_cachep = kmem_cache_create("blkdev_requests",
 			sizeof(struct request), 0, SLAB_PANIC, NULL);
 
-	blk_requestq_cachep = kmem_cache_create("blkdev_queue",
+	blk_requestq_cachep = kmem_cache_create("request_queue",
 			sizeof(struct request_queue), 0, SLAB_PANIC, NULL);
 
 	return 0;

